carbonmark/package.json

```diff
@@ -10,7 +10,7 @@
     "lingui:compile": "lingui compile",
     "lingui:extract:dev": "node ../scripts/lingui.js extract --clean --overwrite --locale=en && node ../scripts/lingui.js extract --clean --overwrite --locale=en-pseudo",
     "postinstall": "npm run lingui:compile",
-    "generate:types": "tsx ./scripts/fetch-api-schema.ts && kubb"
+    "generate:types": "tsx ./scripts/fetch-api-schema.ts & kubb && cp lib/api/client.ts .generated/carbonmark-api-sdk"
```

Comment:
Did you change `&&` into `&` on purpose there?

Reply:
I did yea, we need to wait for the schema fetch before running the generator. This didnt cause any issues yet but it could run the gen on an old schema (possibly)